docs/user-guide.md

```diff
@@ -364,6 +365,39 @@ services:
       - db-data:/var/lib/postgresql/data
 ```
 
+- `kompose.controller.type` defines which controller type should convert for this service
+
+For example:
+
+```
+web:
+  image: wordpress:4.5
+  ports:
+    - '80'
+  environment:
+    WORDPRESS_AUTH_KEY: changeme
+    WORDPRESS_SECURE_AUTH_KEY: changeme
+    WORDPRESS_LOGGED_IN_KEY: changeme
+    WORDPRESS_NONCE_KEY: changeme
+    WORDPRESS_AUTH_SALT: changeme
+    WORDPRESS_SECURE_AUTH_SALT: changeme
+    WORDPRESS_LOGGED_IN_SALT: changeme
+    WORDPRESS_NONCE_SALT: changeme
+    WORDPRESS_NONCE_AA: changeme
+  restart: always
+  links:
+    - 'db:mysql'
+db:
+  image: mysql:5.7
+  environment:
+    MYSQL_ROOT_PASSWORD: password
+  restart: always
+  labels:
+    project.logs: /var/log/mysql
+    kompose.controller.type: daemonset
+```
+
+Service `web` will be converted to `Deployment` as default, service `db` will be converted to `StatefulSet` because of `kompose.controller.type` label.
```

Comment:
this is totally wrong... if you specify kompose.controller.type daemonset, this should be converted to daemonset... if you supply statefulset, this should be converted to statefulset..